website/tests/acceptance/sidebar-test.js

```diff
@@ -1,10 +1,12 @@
 import { module, test } from 'qunit';
-import { fillIn, visit } from '@ember/test-helpers';
+import { fillIn, visit, pauseTest } from '@ember/test-helpers';
```

Comment:
Linter doesn't like this remnant `pauseTest`

Reply:
🤦‍♂️